report_compassion/report/bvr_sponsorship.xml

```diff
@@ -78,10 +78,9 @@
                     <t t-foreach="o.get_months(months, docs.get(o))" t-as="date">
                         <t t-call="report_compassion.report_sponsorship_single_slip">
                             <t t-set="margin" t-value="0"/>
-                            <t t-set="date_split" t-value="date.split(' - ')"/>
                             <t t-set="reference" t-value="o.format_ref"/>
                             <t t-set="communication"
-                               t-value="o.get_communication(date_split[0], date_split[-1], docs.get(o))"/>
+                               t-value="o.get_communication(date_start, date_stop, docs.get(o))"/>
                             <t t-set="amount" t-value="False"/>
```

Comment:
I'm really not sure about this but it crashes telling me the object 'date' doesn't exist

Reply:
It looks good for me